src/js/components/Index.js

```diff
@@ -20,10 +26,13 @@ const VIEW_COMPONENT = {
 
 export default class Index extends Component {
 
-  constructor () {
-    super();
+  constructor (props) {
```

Comment:
now that we added props this would not work anymore: `const Index = new Index(props, context)`;

Can we change this constructor to include `context` ?

`constructor (props, context) {`